### PR TITLE
Build pacaur manually

### PR DESCRIPTION
**Pull request by pavanjadhaw**

### CODE REVIEW

**Comment by pavanjadhaw** [comment]
This pull request fixes following issue, https://github.com/akhilnarang/scripts/issues/32

**Comment by pavanjadhaw** [comment]
@akhilnarang I think we would also need to add a check to see if pacaur is installed or not.
If its installed we can skip this step overall.

**Comment by akhilnarang** [comment]
Yeah cool
Sorry for the late response
Will you do it? Else I'll see later
I'll test and merge this soon

**Comment by pavanjadhaw** [comment]
Yeah it will be easier to do.
I will do necessary changes and notify you.  

Thanks a lot for your work, I always use these scripts on new system to setup environment. :)

**Comment by pavanjadhaw** [comment]
scrot-focused-03-01-18_10-58-11
  

@akhilnarang this should work now, I checked the first part  
but I cant check the later one. So test these changes once you have time
and make changes accordingly (hinting - remove those stupid comments).

File: setup/arch-manjaro.sh [hunk 1]
```diff
@@ -6,8 +6,37 @@ clear
 echo Installing Dependencies!
 # Update
 sudo pacman -Syyu
-# Install pacaur
-sudo pacman -S pacaur
+
+# check if pacaur is installed
+if pacman -Qi $package &> /dev/null; then
```
Comment (akhilnarang) [inline setup/arch-manjaro.sh]: `$package` -> `pacaur`

File: setup/arch-manjaro.sh [hunk 2]
```diff
@@ -6,8 +6,37 @@ clear
...
+    gpg --recv-keys --keyserver hkp://pgp.mit.edu 1EB2638FF56C0C53
+    curl -o PKGBUILD https://aur.archlinux.org/cgit/aur.git/plain/PKGBUILD?h=cower
+    makepkg -i PKGBUILD --noconfirm
+
+    # next install pacaur
+    curl -o PKGBUILD https://aur.archlinux.org/cgit/aur.git/plain/PKGBUILD?h=pacaur
+    makepkg -i PKGBUILD --noconfirm
+
+    # cleanup build directory
+    cd && rm -r ~/tmp
```
Comment (akhilnarang) [inline setup/arch-manjaro.sh]: `cd -;`
`rm -rf ~/tmp`

**Comment by krasCGQ** [comment]
Just a little suggestion,
As some (if not all) Arch derivatives with shipped with their own repositories, it might be better to do these instead:
* Check for availability of `pacaur` in default repositories with `pacman -Ss pacaur`.
* If `pacaur` is exist on one of the default repositories, install with `pacman`, otherwise build from AUR.

**Comment by akhilnarang** [comment]
Script has been updated, dosen't use pacaur now

**Pull request closed without merging**